core/whistle_services-1.0.0/src/services/wh_service_limits.erl

```diff
@@ -27,18 +27,44 @@ reconcile(Services) ->
             lager:debug("unable to get current limits in service: ~p", [_R]),
             Services;
         {'ok', JObj} ->
-            Routines = [fun(S) ->
-                                Quantity = wh_json:get_integer_value(<<"twoway_trunks">>, JObj, 0),
-                                wh_services:update(<<"limits">>, <<"twoway_trunks">>, Quantity, S)
-                        end
-                        ,fun(S) ->
-                                 Quantity = wh_json:get_integer_value(<<"inbound_trunks">>, JObj, 0),
-                                 wh_services:update(<<"limits">>, <<"inbound_trunks">>, Quantity, S)
-                         end
-                         ,fun(S) ->
-                        	 Quantity = wh_json:get_integer_value(<<"outbound_trunks">>, JObj, 0),
-                                 wh_services:update(<<"limits">>, <<"outbound_trunks">>, Quantity, S)
-                         end
-                       ],
+            Routines = [
```

Comment:
Why aren't you calling `reconcile(Services, JObj)` here instead of repeating yourself?